tests/util.py

```diff
@@ -74,10 +74,11 @@ def _redirect_stdout(self, to_file_descriptor):
         sys.stdout = io.TextIOWrapper(os.fdopen(self._stdout_file_descriptor, "wb"))
 
 
-def main(test_dir):
+def main(test_dir, config=None):
```

Comment:
Does this ever get called without `config`? If not, I suggest removing the default `None` behavior.

Reply:
It does sometimes, but I agree it is probably better to be explicit, so I made it a required argument.  I cleaned up the `test_get_time.py` and `test_tracer_metadata.py` scripts so they make use of the `main` function as well.